bentoml/_internal/yatai_rest_api_client/schemas.py

```diff
@@ -164,6 +165,9 @@ class BentoManifestSchema:
     runners: Optional[List[BentoRunnerSchema]] = attr.field(factory=list)
 
 
+TransmissionStrategy = Literal["presigned_url", "proxy"]
```

Comment:
To fit with runtime type checking for attrs.
```suggestion
if TYPE_CHECKING:
    TransmissionStrategy = t.Literal["presigned_url", "proxy"]
else:
    TransmissionStrategy = str
```